### PR TITLE
Configuration instructions on readme

### PR DESCRIPTION
**Pull request by digaoddc**
A better description and configuration example .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,18 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
     end
     ```
 
+## Configuration with env vars
+Insert this in your Phoenix configuration files. Eg:  `config.exs`
+```elixir
+config :powerdnsex, url: {:system, "POWERDNS_URL"},
+                    token: {:system, "POWERDNS_TOKEN"}
+```
+Make sure you set those envinronment variables.
```
Comment (thehunmonkgroup) [inline README.md]: envinronment -> environment

File: README.md [hunk 2]
```diff
@@ -22,8 +22,18 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
     end
     ```
 
+## Configuration with env vars
+Insert this in your Phoenix configuration files. Eg:  `config.exs`
```
Comment (thehunmonkgroup) [inline README.md]: Will this config work in non-Phoenix setups? If so, maybe this should read: "Insert this in your Mix or Phoenix configuration files"

Comment (oborba) [reply README.md]: I'd prefer to leave this as is unless anyone can confirm that the config actually works in a non-Phoenix setup.

Comment (thehunmonkgroup) [reply README.md]: Confirmed as working in a regular Mix project set up via ```mix new```.

I might also recommend showing the alternate configuration of placing the data directly in the config file:
```elixir
config :powerdnsex, url: "localhost:8081",
                    token: "sometoken"
```
This is less complex and would work fine -- I could see possibly using that style and placing the config file via server configuration management software, such as Salt/Chef/Puppet/Ansible.

File: README.md (hunk 1 above)
Comment (digaoddc) [reply README.md]: Great, I'll remove Phoenix reference from the description. And fix this typo

File: README.md (hunk 2 above)
Comment (digaoddc) [reply README.md]: Ok, I will leave both ways to configure PowerDNSex in README, so people can choose what is best for them.

I particularly prefer using env vars, which follows 12 Factor App, and makes your life easier when you have to change your environment.

Comment (thehunmonkgroup) [reply README.md]: Using Salt (and I'm guessing most others of its kind), it's trivial to manage the config file differently based on environment. I can't even imagine managing servers 'manually' these days :)

Comment (digaoddc) [reply README.md]: Yeah, I think so. Anyway, having two ways to configure should be good.

**Pull request merged**